src/software/backend/simulator_backend.h

```diff
@@ -1,15 +1,20 @@
 #pragma once
 
-#include <future>
+#include <atomic>
 #include <mutex>
+#include <thread>
 
 #include "software/backend/backend.h"
 #include "software/backend/simulation/physics/physics_simulator.h"
+#include "software/multithreading/thread_safe_buffer.h"
 #include "software/world/world.h"
 
 /**
  * This class implements the Backend interface using a physics simulation to
  * execute received commands and provide new data to the system.
+ *
+ * This class is intentionally not registered in the BackendFactory since it does
+ * not have a default constructor and has a relatively specialized use case.
```

Comment:
Whoops this was already done, just didn't update the comment.

What would the reasonable default be for simulation speed then? Because it _is_ a simulation backend I would assume the "as fast as possible" mode would be the default. Does that make sense?

Reply:
I think a reasonable default would be realtime, if I'm running the simulator I don't expect faster then realtime, and I feel like intuitively that's true for most people.

Reply:
fixed